test/verify/check-system-realms

```diff
@@ -397,11 +397,16 @@ class CommonTests:
 
         def do_test(authopts, expected, not_expected=None, session_leader=None):
             m.start_cockpit(tls=True)
-            output = m.execute(['curl', '-ksS', '-D-', *authopts, 'https://localhost:9090/cockpit/login'])
-            for s in expected:
-                self.assertIn(s, output)
-            for s in (not_expected or []):
-                self.assertNotIn(s, output)
+
+            def try_auth():
+                output = m.execute(['curl', '-ksS', '-D-', *authopts, 'https://localhost:9090/cockpit/login'])
+                for s in expected:
+                    self.assertIn(s, output)
+                for s in (not_expected or []):
+                    self.assertNotIn(s, output)
+                return True
+
+            testlib.wait(try_auth, delay=5, tries=10)
```

Comment:
Could the test do that retry the first time only? We want to treat that as "async init"/"warming up". We totally don't expect this to be wonky afterwards, that could hide bugs in our own code as well.

Reply:
Yes, indeed.